spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -32,10 +32,10 @@
       it "rounds fee calculations correctly" do
         # $0.86 + 20% = $1.032
         # Fractional cents should be immediately rounded down and not carried through
-        expect(page).to have_selector '.cart-item-price',         text: '$1.03'
-        expect(page).to have_selector '.cart-item-total',         text: '$8.24'
-        expect(page).to have_selector '.order-total.item-total',  text: '$8.24'
-        expect(page).to have_selector '.order-total.grand-total', text: '$8.24'
+        expect(page).to have_selector '.cart-item-price',         text: "#{@currency_symbol}1.03"
```

Comment:
We should use that helper. And probably will have to write one for Angular front-end to unify (if there is none). Basically will have to go through each failing test and check what is happening with each method. Just please give me more time for that :)

As for `money` helper in specs I would suggest it is a bit too short and doesn't say what it does, just something with money, it is like `thing(1.03)` and you still have to look what it does in the code (which is not a bad thing). `display_money` would mean that it gives output, right? Maybe `text: with_currency(1.03)` would be more meaningful? Apologies if github's PR comments is not a good place for discussions like this.